docs/contributing/gsoc/2022/index.rst

```diff
@@ -165,7 +165,7 @@ Most of our communication takes place on the
 We run a weekly meeting that we encourage everyone to join. We get people
 started and do debugging.
 
-If we're not responding, we may be busy and forgotten about your message. Ping
+If we're not responding, we may be busy and have forgotten about your message. Ping
```

Comment:
Wording changes should be done in a separate PR

Reply:
Aah sure
Thanks for the review, will keep in mind for the future